pycheribuild/projects/project.py

```diff
@@ -1711,7 +1711,7 @@ def default_compiler_flags(self):
                            # "-Xclang", "-cheri-bounds=everywhere-unsafe"])
                            ])
         # Add mxcaptable for projects that need it
-        if self.compiling_for_cheri() and self.config.cheri_cap_table_abi != "legacy":
+        if self.compiling_for_cheri():
```

Comment:
This really was to be guarded by MIPS... are we not getting `clang-10: warning: argument unused during compilation: '-mxcaptable' [-Wunused-command-line-argument]` left right and centre on RISC-V? Maybe because we don't set the `force_*_linkage` very often?

Reply:
Yeah I think it's only WebKit and PostgreSQL when statically linked.